alpha/lib/model/entry.php

```diff
@@ -3487,9 +3488,13 @@ public function shouldCloneByProperty($property)
 				$currentOption = $cloneOption->getItemType();
 				$currentType = $cloneOption->getRule();
 
-				if ($currentOption ==  $property && $currentType == CloneComponentSelectorType::EXCLUDE_COMPONENT)
+				if ($currentOption ==  $property)
 				{
-					$cloneCategories = false;
+					if ($currentType == CloneComponentSelectorType::EXCLUDE_COMPONENT)
+						$cloneCategories = false;
```

Comment:
instead of setting $cloneCategories, you could simply return false, then you won't need else, break, setting to default at the beginning of the method.